WordPress/Classes/ViewRelated/Reader/Comments/ReaderCommentsFollowPresenter.swift

```diff
@@ -132,9 +132,11 @@ class ReaderCommentsFollowPresenter: NSObject {
     // MARK: - Notification Sheet
 
     @objc func showNotificationSheet(sourceBarButtonItem: UIBarButtonItem?) {
-        let sheetViewController = ReaderCommentsNotificationSheetViewController(isNotificationEnabled: post.receivesCommentNotifications, delegate: self)
-        let bottomSheet = BottomSheetViewController(childViewController: sheetViewController)
-        bottomSheet.show(from: self.presentingViewController, sourceBarButtonItem: sourceBarButtonItem)
+        showNotificationSheet(sourceBarButtonItem: sourceBarButtonItem)
+    }
+
+    func showNotificationSheet(sourceView: UIView?) {
+        showNotificationSheet(sourceView: sourceView)
```

Comment:
This recursively calls the method and caused the app to go into an infinite loop. Strangely enough, `@objc showNotificationSheet(sourceBarButtonItem:)` works just fine – perhaps due to the `@objc` attribute? 🤔 

Some options:
1. Remove the default parameter values from the private methods, so the public method explicitly calls the private method with two parameters.
2. Slightly change the private method name.
3. Make the private method public, but we'd also need to add `@objc` to it.

Reply:
Um... yea... the private method name was different. Until I decided to make it the same, and then clearly didn't retest. :facepalm: I have renamed it once again. Sorry about that!